docs/detections/visual-event-analyzer.asciidoc

```diff
@@ -2,46 +2,52 @@
 [role="xpack"]
 == Visual event analyzer
 
-Elastic Security allows any event detected by Elastic Endpoint to be analyzed using a process-based visual analyzer. This enables security analysts to drill down into all related hosts, processes, and other events to aid in their investigations.
+{elastic-sec} allows any event detected by {elastic-endpoint} to be analyzed using a process-based visual analyzer, which shows a graphical timeline of processes that led up to the alert and the events that occurred immediately after. Viewing events in the visual event analyzer is useful to determine the origin of potentially malicious activity and other areas in your environment that may be compromised. It also enables security analysts to drill down into all related hosts, processes, and other events to aid in their investigations.
 
 [float]
 [[find-events-analyze]]
 === Find events to analyze
 
-You can only visualize events triggered by hosts configured with the Elastic Endpoint Security Integration or any sysmon data from `winlogbeat`.
+You can only visualize events triggered by hosts configured with the {endpoint-security} integration or any `sysmon` data from `winlogbeat`.
```

Comment:
The `{endpoint-security}` attribute isn't rendering correctly in the HTML output. Should it be `{es-sec-endpoint}` instead?

Reply:
Thanks - it's `endpoint-sec`. Fixed.